### PR TITLE
Updated chart & table divs with some styling for aesthetics

### PR DESCRIPTION
**Pull request by jpjsonic**

### CODE REVIEW

File: viewer/src/components/items/Table.js [hunk 1]
```diff
@@ -67,6 +64,7 @@ const Table = ({ table, project, itemWidth, height, width }) => {
     }
 
     return (
+        <ItemContainer>
```
Comment (tgsoverly) [inline viewer/src/components/items/Table.js:67]: Might want this inside the provider

File: viewer/package.json [hunk 2]
```diff
@@ -34,6 +34,7 @@
     "react-router-dom": "^6.21.0",
     "react-scripts": "5.0.1",
     "react-select": "^5.8.0",
+    "styled-components": "^6.1.12",
```
Comment (tgsoverly) [inline viewer/package.json:37]: This did look used.

**Review by tgsoverly — APPROVED** [review]
Just a couple thoughts

File: viewer/src/components/items/Table.js (hunk 1 above)
Comment (jpjsonic) [reply viewer/src/components/items/Table.js:67]: What do you mean by inside the provider?

File: viewer/package.json (hunk 2 above)
Comment (jpjsonic) [reply viewer/package.json:37]: Oh, yup. I was maybe going to use it, but went with the tailwind approach instead.

File: viewer/src/components/items/Table.js (hunk 1 above)
Comment (tgsoverly) [reply viewer/src/components/items/Table.js:67]: <Provider>
  <ItemContainer/>
</Provider>

**Comment by jpjsonic** [comment]
image

**Pull request merged**